README.md

```diff
@@ -44,83 +44,223 @@ To get started with CrewAI, follow these simple steps:
 
 ### 1. Installation
 
+Ensure you have Python >=3.10 <=3.13 installed on your system. CrewAI uses [Poetry](https://python-poetry.org/) for dependency management and package handling, offering a seamless setup and execution experience.
+
+First, if you haven't already, install Poetry:
+
+```bash
+pip install poetry
+```
+
+Then, install CrewAI:
+
 ```shell
 pip install crewai
 ```
 
-If you want to install the 'crewai' package along with its optional features that include additional tools for agents, you can do so by using the following command: pip install 'crewai[tools]'. This command installs the basic package and also adds extra components which require more dependencies to function."
+If you want to install the 'crewai' package along with its optional features that include additional tools for agents, you can do so by using the following command:
 
 ```shell
 pip install 'crewai[tools]'
 ```
+The command above installs the basic package and also adds extra components which require more dependencies to function.
+
+Lock the dependencies and install them by using the CLI command:
+
+```shell
+crewai install
```

Comment:
I think this needs to be below crewai create command??

Reply:
Good point. Totally missed that.

Reply:
Updated!